### PR TITLE
FAT-987

### PR DESCRIPTION
**Pull request by viktor-draban**
link to my [ticket](https://issues.folio.org/browse/FAT-987)
create automateFeeFines.feature

### CODE REVIEW

File: mod-feesfines/src/main/resources/domain/mod-feesfines/features/automateFeeFines.feature [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+    Given path 'instance-types'
+    And request requestInstanceTypeEntity
+    When method POST
+    Then status 201
+
+    * def requestContributorNameTypeEntity = read('samples/instance-type-request-entity.json')
+    Given path 'contributor-name-types'
+    And request requestContributorNameTypeEntity
+    When method POST
+    Then status 201
```
Comment (alexanderkurash) [inline mod-feesfines/src/main/resources/domain/mod-feesfines/features/automateFeeFines.feature:44]: Do we need both?

File: mod-feesfines/src/main/resources/domain/mod-feesfines/features/automateFeeFines.feature [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+    * def checkOutByBarcodeId = call uuid1
+    * def overdueFinePoliciesId = call uuid1
+    * def lostItemFeePolicyId = call uuid1
+    * def loanPolicyId = call uuid1
+    * def patronPolicyId = call uuid1
+    * def requestPolicyId = call uuid1
+
+  Scenario: verify Account.contributors field
+    # instance and its prerequisites
+    * def requestInstanceTypeEntity = read('samples/instance-type-request-entity.json')
```
Comment (alexanderkurash) [inline mod-feesfines/src/main/resources/domain/mod-feesfines/features/automateFeeFines.feature:34]: request -> end (instanceTypeEntityRequest, instance-type-entity-request.json)

File: mod-feesfines/src/main/resources/domain/mod-feesfines/features/automateFeeFines.feature [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+
+    * def requestLocationEntity = read('samples/location/location-request-entity.json')
+    Given path 'locations'
+    And request requestLocationEntity
+    When method POST
+    Then status 201
+
+
+    # holding
+    * def requestHoldingEntity = read('samples/holding-request-entity.json')
```
Comment (alexanderkurash) [inline mod-feesfines/src/main/resources/domain/mod-feesfines/features/automateFeeFines.feature:86]: holdings

File: mod-feesfines/src/main/resources/domain/mod-feesfines/features/automateFeeFines.feature [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+    Given path 'holdings-storage', 'holdings'
+    And request requestHoldingEntity
+    When method POST
+    Then status 201
+
+
+    # item and its prerequisites
```
Comment (alexanderkurash) [inline mod-feesfines/src/main/resources/domain/mod-feesfines/features/automateFeeFines.feature:93]: Reformat

File: mod-feesfines/src/main/resources/domain/mod-feesfines/features/automateFeeFines.feature [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+    When method POST
+    Then status 201
+
+    * def requestPolicy = read('samples/policies/request-policy-request-entity.json')
+    Given path 'request-policy-storage/request-policies'
+    And request requestPolicy
+    When method POST
+    Then status 201
+
+    * def priorityVariable = 'priority: t, s, c, b, a, m, g fallback-policy: l ' + loanPolicyId + ' o ' + overdueFinePoliciesId + ' i ' + lostItemFeePolicyId + ' r ' + requestPolicyId + ' n ' + patronPolicyId
```
Comment (alexanderkurash) [inline mod-feesfines/src/main/resources/domain/mod-feesfines/features/automateFeeFines.feature:143]: Rename

File: mod-feesfines/src/main/resources/domain/mod-feesfines/features/automateFeeFines.feature [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+    Then status 201
+
+    * def feefineTypeRequestEntity = read('samples/feefine-request-entity.json')
+    * feefineTypeRequestEntity.automatic = true
+    * feefineTypeRequestEntity.feeFineType = "Overdue fine"
+    * feefineTypeRequestEntity.defaultAmount = null
+    Given path 'feefines'
+    And request feefineTypeRequestEntity
+    When method POST
+    Then status 201
```
Comment (alexanderkurash) [inline mod-feesfines/src/main/resources/domain/mod-feesfines/features/automateFeeFines.feature:165]: Separate group

File: mod-feesfines/src/main/resources/domain/mod-feesfines/features/automateFeeFines.feature (hunk 1 above)
Comment (viktor-draban) [reply mod-feesfines/src/main/resources/domain/mod-feesfines/features/automateFeeFines.feature]: we need just contributor Id, we may mock it and  omit creating this entity, but I've already faced tons of situation when mock Id doesn't work for some complicated computing, so I suggest stay it as now.

File: mod-feesfines/src/main/resources/domain/mod-feesfines/features/automateFeeFines.feature (hunk 2 above)
Comment (viktor-draban) [reply mod-feesfines/src/main/resources/domain/mod-feesfines/features/automateFeeFines.feature]: done

**Pull request merged**